### DISCUSSION TITLE
CONTRIBUTIONS IN PRIVATE REPOSITORIES OF ORGANIZATIONS ARE NOT SHOWN IN MY PROFILE

### DISCUSSION THREAD

**Discussion by Dymidless** [community/community#66346] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Hey. My organization is called Meteor and there I have made several commits to private repositories. The problem is that none of them are shown on my profile. What I can do?

**Comment by fathi-riham-mn** [comment, marked as answer]
Check Your Profile Settings:

Go to your GitHub profile settings by clicking on your profile picture in the top right corner, then selecting "Settings."
In the left sidebar, click on "Contributions."
Ensure that the option "Include private contributions on my profile" is enabled. If it's disabled, private contributions won't appear on your profile.
Verify Repository Visibility:

Make sure that the private repositories you've contributed to are indeed set as "private" repositories in your organization. If a repository is mistakenly set as "public," contributions to it won't count as private contributions.
Repository Ownership:

Check if the repositories you've contributed to are owned by your organization "Meteor." If the repositories are owned by another organization or user, your contributions may not be counted on your profile.
Commit Email Address:

Ensure that the email address associated with your GitHub account matches the email address used in your commit authorship. GitHub uses the email address in the commit to attribute contributions to your profile.
Cache and Timing:

Sometimes, contributions take a little while to appear on your profile due to caching. Wait for some time and check if the contributions appear after a while.
Contact GitHub Support:

If the issue persists despite checking all settings and verifying ownership, consider reaching out to GitHub Support. They can investigate the issue and help you resolve it.

**Reply by danishusf** [reply]
for anyone wondering.

open terminal and execute:

git config user.email "add private email from settings"

git config user.name "your github username"

this made it work.

**Comment by Xavopls** [comment]
For future reference, please note that the "Contributions" section has been relocated from the left sidebar to within the "Public profile" section.

**Comment by WHITELOTUS0** [comment]
Hi there,

Contribution counts on GitHub are specifically tied to certain actions and branches. For a commit to appear on your profile, it generally needs to be made to the repository's default branch or the `gh-pages` branch.

Here's a summary of what counts as a contribution:

- Committing to a repository's default branch or `gh-pages` branch
- Creating a branch
- Opening an issue
- Opening a discussion
- Answering a discussion
- Proposing a pull request
- Submitting a pull request review

If your commits are made to a branch other than the default branch, they will only show up on your profile if they are merged into the default branch. For more details on what counts as a contribution and how to manage your profile, you can refer to GitHub's [documentation on contributions](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/viewing-contributions-on-your-profile#what-counts-as-a-contribution).

I hope this helps clarify the situation.

**Comment by RushilJalal** [comment]
@fathi-riham-mn Hey. I just made a commit to a private repo made by me but it doesn't show up on my contributions graph. I have enabled the option "Include private contributions on my profile". Kindly help. Thanks.

**Reply by Usman-Click** [reply]
Me too, though I forked both the repos
Did that implies anything?

**Reply by RushilJalal** [reply]
Read the marked answer above.

**Comment by OlivierJDI** [comment]
I am sorry but this is such a mess. Displaying contributions should be super intuitive.
